chat/v1/messages.proto

```diff
@@ -227,6 +228,14 @@ message Content {
   }
 }
 
+// Object representing a reaction.
+message Reaction {
+  // Emote data for this reaction.
+  emote.v1.Emote emote = 1;
```

Comment:
A comment needs to be added that an emote's image id is used as the "identifier" for the reaction. Names should be deduplicated by suffixing them with `~number`.